ieeg_motorMiller2007/sub-bp/ses-01/ieeg/sub-bp_ses-01_space-T1w_electrodes.tsv

```diff
@@ -0,0 +1,48 @@
+name	x	y	z	size	type	manufacturer
+1	-38.2367221940641	42.990911138808	32.1161296015272	2.3	surface	AdTech
```

Comment:
I should correct the size to surface area (currently diameter)

Reply:
While you are at fixing your own dataset, could you correct the epilepsy one as well to have "Other" instead of "T1w" (as you noted above)? :-)

Reply:
yes! I should make a pull request to your fork/branch, correct? (will be tomorrow)

Reply:
yes, to https://github.com/sappelhoff/bids-examples/tree/bep010_ieeg would be ideal, thanks!

Reply:
just made a pull request to your fork/ieeg branch!